src/pinns_pde_solve.jl

```diff
@@ -335,28 +363,29 @@ function build_symbolic_loss_function(eqs,indvars,depvars,
     expr_loss_function = :(($vars) -> begin $ex end)
 end
 
-function build_loss_function(eqs,_indvars,_depvars, phi, derivative,initθ;bc_indvars=nothing)
+function build_loss_function(eqs,_indvars,_depvars, phi, derivative,initθ;bc_indvars=nothing,eq_params=SciMLBase.NullParameters(),param_estim=false,default_p=nothing)
     # dictionaries: variable -> unique number
     depvars,indvars,dict_indvars,dict_depvars = get_vars(_indvars, _depvars)
     bc_indvars = bc_indvars==nothing ? indvars : bc_indvars
     return build_loss_function(eqs,indvars,depvars,
                                dict_indvars,dict_depvars,
                                phi, derivative,initθ,
-                               bc_indvars = bc_indvars)
+                               bc_indvars = bc_indvars, eq_params=eq_params,param_estim=param_estim,default_p=default_p)
 end
 
 function build_loss_function(eqs,indvars,depvars,
```

Comment:
`build_loss_function` should be RuntimeGeneratedFunction's wrapper for `build_symbolic_loss_function` and have the same parameter for both.

Reply:
> @ashutosh-b-b see [#256 (comment)](https://github.com/SciML/NeuralPDE.jl/pull/256#discussion_r589249132)



> @ashutosh-b-b see [#256 (comment)](https://github.com/SciML/NeuralPDE.jl/pull/256#discussion_r589249132)

Yes I changed them to have the same parameters.